src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java

```diff
@@ -35,7 +35,9 @@
     private static final Logger logger = LogManager.getLogger(ExtractCohort.class);
     private ExtractCohortEngine engine;
 
-   @Argument(
+    public enum FilteringType { NONE, GENOTYPE, SITES};
```

Comment:
does selecting GENOTYPE also imply that we are doing SITE level filtering? So maybe the enum should be {NONE, SITES_ONLY, FULL} (or instead of FULL maybe SITES_AND_GT)?